core/frontend/src/components/wizard/Wizard.vue

```diff
@@ -34,18 +34,43 @@
 
         <v-stepper-items>
           <v-stepper-content step="0">
-            <RequireInternet @next="nextStep()" />
+            <v-card class="ma-2 pa-5">
+              <div class="welcome" style="display: flex; justify-content: center; align-items: center; width: 100%;">
+                Welcome to BlueOS!
+              </div>
+              Welcome to BlueOS!
+              In this setup wizard we will guide you through the initial setup of your vehicle.
```

Comment:
...initial configuration of your vehicle...

Just so we don't use "setup" three times in a row.